src/ppx/ppx_eliom_client.ml

```diff
@@ -290,5 +292,8 @@ end
 include Make(Pass)
 
 let () =
-  Migrate_parsetree.Driver.register ~name:"ppx_eliom_client" ~args:driver_args
-    Migrate_parsetree.Versions.ocaml_408 mapper
+  Ppxlib.Driver.register_transformation
+    ~preprocess_impl:(fun str -> mapper.AM.structure mapper str)
+    ~preprocess_intf:(fun sig_ ->
```

Comment:
Why do you need `preprocess_impl`/`preprocess_intf`? Do the three `eliom` PPXs need to be applied before other PPXs are applied?